var/spack/repos/builtin/packages/py-dh-scikit-optimize/package.py

```diff
@@ -21,6 +21,11 @@ class PyDhScikitOptimize(PythonPackage):
     homepage = "https://github.com/deephyper/scikit-optimize"
     pypi     = "dh-scikit-optimize/dh-scikit-optimize-0.9.0.tar.gz"
 
+    version('0.9.5', sha256='c2777839a536215ab40fc5db2242809ccffd5e5b23718e23d58ea76ff35a7543')
+    version('0.9.4', sha256='9acfba4077fe45f3854a4af255763a3e8a396c05bd2a7c761a969171366b3840')
+    version('0.9.3', sha256='0c922c61dc1f010c7bbd2f0930c766e8ae040c35b129e4de6d51b611fd72b7c9')
+    version('0.9.2', sha256='9fd2d3accad7c3aca1d0edd0c88ec5490e9bc7a07f31d88f4051466a132dfd1d')
+    version('0.9.1', sha256='70e70011f4fd107082174c557de70ba37c215e71a676788234b85d5c370fd7bd')
     version('0.9.0', sha256='fe70aa57ec5150a3d356b2184f0dda1ecc4ecb7e82d35edac3980094d409d676')
 
     variant('plots', default=False,
```

Comment:
I no longer see configspace listed, but you're missing a dep on pandas too

Reply:
The ConfigSpace dependency was not listed previously either (if I remember our conversations in the original PR).
The pandas dependency seems to have been added starting from 0.9.3

Reply:
I see pandas present in 0.9.1 but not in 0.9.0. I don't see py-configspace anywhere in the 0.9* versions.
I'll add a commit removing configspace and adding pandas as necessary.

Reply:
As ConfigSpace is non optionally required, I would keep it.

Reply:
Ok I removed the corresponding `when=...` statement.